src/JoomlaBrowser.php

```diff
@@ -670,4 +670,45 @@ public function displayModuleOnAllPages($module)
         $I->click(['xpath' => "//div[@id='toolbar-apply']/button"]);
         $I->waitForText('Module successfully saved',30,['id' => 'system-message-container']);
     }
+
+    /**
+     * Creates a menu item with the Joomla menu manager, only working for menu items without additional required fields
+     *
+     * @param   string  $menuTitle     The menu item title
+     * @param   string  $menuCategory  The category of the menu type (for example Weblinks)
+     * @param   string  $menuItem      The menu item type / link text (for example List all Web Link Categories)
+     * @param   string  $menu          The menu where the item should be created
+     */
+    public function createMenuItem($menuTitle, $menuCategory, $menuItem, $menu = "Main Menu")
+    {
+        $I = $this;
+        $I->amOnPage('administrator/index.php?option=com_menus&view=menus');
+        $I->waitForText('Menus', '30', ['css' => 'H1']);
+
+        // Choose the right menu
+        $I->click(['xpath' =>  "//*[@id=\"menuList\"]/tbody/tr/td[2]/a[contains(text(), '" . $menu . "')]"]);;
```

Comment:
I have created this issue to improve this locator when we will have the chance: https://github.com/joomla-projects/joomla-browser/issues/62